Yank/multistate/multistatereporter.py

```diff
@@ -1060,6 +1060,49 @@ def write_last_iteration(self, iteration):
         """
         self._storage_analysis.variables['last_iteration'][0] = iteration
 
+    def read_logZ(self, iteration):
```

Comment:
We don't necessarily want to keep these `read_logZ/write_logZ`, or keep them here if we want to keep them.